build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -8,7 +8,8 @@
   </PropertyGroup>
   <ItemGroup>
     <MonoDocCopyItem Include="monodoc.dll" />
-    <MonoDocCopyItem Include="monodoc.dll.mdb" />
+    <MonoDocCopyItem Include="monodoc.pdb"  Condition="Exists ('monodoc.pdb')" />
```

Comment:
I don't think this `Condition` can *possibly* work. It's going to use the current working directory, and there are no odds that the current working directory will be `external/mono/mcs/class/lib/net_4_x`.

I *think* what you'd actually want to do is remove the `Condition` on this line and the following line, and instead update `@(_MonoDocCopyItems)` -- line 30? -- to be:

```xml
<ItemGroup>
  <_MonoDocCopyItems
      Condition=" Exists ('$(_MonoOutputDir)\%(Identity)') "
      Include="@(MonoDocCopyItem->'$(_MonoOutputDir)\%(Identity)')"
  />
  <_MonoDocInstalledItems
      Condition=" Exists ('$(_MonoOutputDir)\%(Identity)') "
      Include="@(MonoDocCopyItem->'$(_MandroidDir)\%(Identity)')"
  />
</ItemGroup>
```

@dellis1972: Does the above sound correct to you?